proto/sf/arweave/type/v1/type.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package cs.arweave.codec.v1;
+package sf.arweave.type.v1;
 
 option go_package = "github.com/streamingfast/sf-ethereum/pb/sf/ethereum/codec/v1;pbcodec";
```

Comment:
You need to update the package name here, so that project that will depend on that generates the correct dependant Go package.